guide/terms.md

```diff
@@ -9,44 +9,81 @@ consistently across the localization of the entire product.
 ```
 -brand-name = Firefox
 
-app-title = { -brand-name }
-has-updated = { -brand-name } has been updated.
+about = About { -brand-name }.
+update-successul = { -brand-name } has been updated.
 ```
 
-## Terms and Variants
+## Parameterized Terms
 
-Values of terms may have multiple [variants](variants.html), including the
-default variant marked with an asterisk (`*`). For instance, variants may
-correspond to different grammatical cases.
+Term values follow the same rules as message values. They can be simple text,
```

Comment:
I'm quite lost in this paragraph. For example, I don't understand the `terms can only reference variables which have been explicitly defined` part. I'm likely oversimplifying things, but:

```
Term values follow the same rules as message values. They can be simple text, or
they can interpolate other expressions, including variables. However, while
messages will receive data for variables directly from the app, terms will
receive such data from the message referencing them. Such references take the
form of `-term(…)` where the variables available inside of the term are defined
between the parentheses, e.g. `-term(param: "value")`.
```

Reply:
This is indeed much better, thanks!